feat: :sparkles: Added shell directive to fetch and describe relationships

mdarin opened the pull request:
Added shell directives `\rr` to fetch the configured relationship list and `\rl` to describe a single relationship.

naorpeled commented:
Hey @mdarin,
Thanks for this awesome PR—you rock! 😎

A few thoughts:
- Could we add some tests to validate this? 🙏
- I'm thinking we should remove `\rl all`, users will likely search for specific relationship types. What do you think?
- The "relationship not found" message seems a bit unclear. Perhaps we can make it more specific? Here's what I'm seeing:
![image](https://github.com/user-attachments/assets/a3a20c87-edd4-4337-b790-2d9832367610)
- Let's avoid adding a new dependency and use utility functions instead.

mdarin commented:
@naorpeled Hi!
Thanks for this review and your remarks! 🤝 
Okay, I will remake it soon.

mdarin commented:
* Removed `\rl all` (useless)
* `lodash` functions incapsulated into the project (core/utils.go)
* Created command actions `list` and `describe` 
* Added test for both actions 
* Error message is more clear now

AvitalTamir commented:
@mdarin what an incredible contribution, and beautiful PR!
Thank you so much for this one, looking forward to including this in our next release.

@naorpeled kudos for the thoughtful code review 🙌 
This is approved on my end, feel free to merge if you have no further comments.

naorpeled reviewed (approved):
🔥

naorpeled commented:
LGTM!
Once the CI is green I'll merge this 😎

mdarin commented:
@AvitalTamir Thank you a lots! 

@naorpeled All checks have passed! ✅

naorpeled commented:
Thanks @mdarin!
